tests/analyze-fixtures/ConstantNaming.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\Compiling\Statements;
+
+class ConstantNaming
+{
+    const UPPERCASE = 2;
+    const Lowercase = 1;
```

Comment:
Could you add a case when several constants are declared using a single `const` keyword?

```php
const Foo = 1, Bar = 2;
```